app/views/microcosms/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :heading do %>
+  <h1><%= t(".title") %></h1>
+  <ul class='secondary-actions clearfix'>
+    <li><%= link_to image_tag("new.png", :class => "small_icon", :border => 0) + t(".new"), new_microcosm_path, :title => t(".new_title") %></li>
+  </ul>
+<% end %>
+
+<% if @microcosms_i_organize %>
+  <%= render :partial => "index_list", :locals => { :microcosms => @microcosms_i_organize, :header => t(".microcosms_i_organize") } %>
+<% end %>
```

Comment:
I just spotted the All Traces / My Traces design pattern.  I'll apply that here.